### DISCUSSION TITLE
[BP#98] Cas de CDN contre-productifs ?

### DISCUSSION THREAD

**Discussion by aw73** [cnumr/best-practices#343] [Bonnes pratiques] [answered]
Il s'agit d'une question.

Dans le cadre d'un site à faible trafic (< 100 visites par jour) et / ou consulté à l'échelle locale (France par exemple), l'usage d'un CDN n'est-il pas contre-productif ?
Quelqu'un a-t-il des éléments de comparaison ?

Si c'est le cas, il pourrait être intéressant d'apporter cette nuance/pécision dans la bonne pratique ?

Source de mon interrogation : https://ichi.pro/fr/quand-eviter-d-utiliser-un-cdn-sur-votre-site-web-217765499967284

**Comment by ACTLEM** [comment, marked as answer]
Merci pour la pertinence de la remarque. Je vais ajouter des nuances/exceptions dans la BP.
